pyacq/dsp/overlapfiltfilt.py

```diff
@@ -390,39 +389,39 @@ def set_params(self, engine, coefficients, nb_channel, dtype, chunksize, overlap
 
 class OverlapFiltfilt(Node,  QtCore.QObject):
     """
-    Node for filtering with forward-backward method (filtfilt).
-    This use sliding overlap technics.
+    Node for filtering with forward-backward method (filtfilt) using second order
+    (sos) coefficient and a sliding, overlapping window.
     
-    The chunksize and the overlapsize are important for the accuracy of filtering.
-    You need to study them carfully, otherwise the result should be the same as a
-    real filtfilt ona long term signal. You must check the residual between real offline filtfitl
-    and this online OverlapFiltfilt.
-    Note that the chunksize have a strong effect on low frequency.
-    
-    This uses Second Order (sos) coeeficient.
-    It internally use scipy.signal.sosfilt which is available only on scipy >0.16
+    It internally uses scipy.signal.sosfilt which is available only on scipy >0.16
     
```

Comment:
@samuelgarcia, check this carefully--I'm not certain I understood the original correctly.